examples/script-tag-dynamic-imports/.gitignore

```diff
@@ -0,0 +1,24 @@
+# build output
+dist/
+# generated types
+.astro/
+
+# dependencies
+node_modules/
+
+# logs
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
```

Comment:
Nitpick: Could be simplified to `*.log*`

Reply:
This is just default from pnpm create astro